repose-aggregator/commons/configuration/src/test/resources/unmarshallerValidator/xsd/httpx-headers.xsd

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
This is the flag for the license-maven-plugin to neither add a header to this file nor fail a build because it doesn't have one. Since this is a file not maintained by Repose, it is ignored for the purpose of this plugin by the build.